app/jobs/reports/drop_off_report.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'reporting/drop_off_report'
+
+module Reports
+  class DropOffReport < BaseReport
+    REPORT_NAME = 'drop-off-report'
+
+    attr_accessor :report_date
+
+    def perform(report_date)
+      return unless IdentityConfig.store.s3_reports_enabled
+
+      self.report_date = report_date
+      subject = "Drop Off Report - #{report_date.to_date}"
+
+      csv = report_maker.to_csv
+
+      save_report(REPORT_NAME, csv, extension: 'csv')
+
+      reports = report_maker
```

Comment:
I think that `ReportMailer.tables_report` expects the reports argument to be an array and in this context it will be one instance of `Reporting::DropOffReport`